generators/mail/templates/common/manifest.xml

```diff
@@ -7,6 +7,7 @@
   <DefaultLocale>en-US</DefaultLocale>
   <DisplayName DefaultValue="<%= projectDisplayName %>" />
   <Description DefaultValue="[Outlook Add-in description]"/>
+  <SupportUrl DefaultValue="http://contoso.com/support" />
```

Comment:
SupportUrl element is only required if the add-in is built to be submitted to the Office Store. Some customers build their add-ins to be used internally only and this won't be necessary. If you could add a comment to it and provide an example such as `<!--<SupportUrl DefaultValue="[Outlook Add-in support url]">-->`, I'd be happy to merge it into the source code.